frontend/packages/core/src/flags.tsx

```diff
@@ -0,0 +1,115 @@
+import * as React from "react";
+import type { clutch as IClutch } from "@clutch-sh/api";
+
+import type { ClutchError } from "./Network/errors";
+import { client } from "./Network/index";
+
+const FF_POLL = +process.env.REACT_APP_FF_POLL || 300000;
```

Comment:
Is there a place we can put `FF_POLL` so we can have a source of truth? I see we set the same const in `frontend/packages/core/src/AppProvider/index.tsx`.

Reply:
Yeah we could have a constants file for core. Good idea